CommandLineOptions.cpp

```diff
@@ -44,6 +44,17 @@ void SetLoggingPriorityOption(std::string opt, std::string val, CommandLineOptio
         opts.logging_opts.min_priority = priority;
 }
 
+void SetPlaygroundOption(std::string opt, std::string val, CommandLineOptions &opts)
+{
+    int is_playground = llvm::StringSwitch<int>(val)
+        .Case("true", 1)
+        .Case("false", 0)
+        .Default(-1);
+    if(is_playground == -1)
+        std::cout << "[Warning] is_playground is neither \"true\" nor \"false\". Defaulting to \"false\"\n";
```

Comment:
log

Reply:
Why not just make this a boolean flag instead?